packages/teams-js/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0-beta.3
+
```

Comment:
I did run yarn changefile, but accidently hit enter for `patch` instead of `major`. It did not create any entry and I thought now I have to do it manually

Reply:
After removing the change, do I need to run yarn changefile again?

Reply:
beachball won't create an entry in the changelog until we release, your changes are safe and sound under the `change` directory :) I'll shoot over a couple links to better explain.

You can also modify the version in the changefile manually.

Reply:
You don't need to run yarn changefile again :)

Reply:
I will remove this. 
Thank you for catching this and sharing beachball links!

Reply:
thanks for being so quick to respond!!